wbooks/src/app/components/AppNavigator/index.tsx

```diff
@@ -55,6 +56,7 @@ function AppNavigator() {
         screenOptions={({ route }) => ({
           tabBarIcon: ({ focused }) => <TabBar focused={focused} route={route} />
         })}>
+        <Tab.Screen name="Login" component={Login} options={{ tabBarVisible: false }} />
```

Comment:
Login is not a member of the tab screen.

Reply:
as maxi says, login should be another screen on the stack

Reply:
Ok, where can I put it?